python/ql/lib/semmle/python/ApiGraphs.qll

```diff
@@ -807,7 +807,7 @@ module API {
      * `lbl` in the API graph.
      */
     cached
-    predicate use(TApiNode base, Label::ApiLabel lbl, DataFlow::Node ref) {
+    predicate use(Node base, Label::ApiLabel lbl, DataFlow::Node ref) {
```

Comment:
At one point in time, at least, the use of `TApiNode` instead of `Node` was intentional, as the latter can be more costly (I think stuff like `toString` gets evaluated on `Node`, but not on `TApiNode`).
Unless you're really sure that this won't affect performance, I would suggest leaving these be.

Reply:
Fair point, I'll remove that bit :+1: